src/wizards/tokenSwapDealWizard/stages/submitStage/submitStage.html

```diff
@@ -137,6 +137,57 @@ <h4 class="submitCardHeader">Terms</h4>
                       <pre class="subtext">${clause.text}</pre>
                     </div>
                   </template>
+                </template>
+
+                <div class="submitContentLabel">DAOplomat Reward</div>
+
+                <div class="submitDaoTokenCard">
+
+                  <pcard class="rewardsCard">
+                    <table class="rewardsTable">
+                      <tr>
+                        <td class="submitContentLabel">
+                          Total DAOplomat Reward
+                        </td>
+                        <td class="submitContentLabel">
+                          Total Value (USD)
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>${submitData.terms.daoplomatRewards.percentage * 100}%</td>
+                        <td>~${submitData.terms.daoplomatRewards.percentage * tokensTotal | currency}</td>
+                      </tr>
+                    </table>
+
+                    <hr>
+
+                    <table class="rewardsTable">
+                      <tr>
+                        <td class="submitContentLabel">
+                          Total DAOplomat Reward
+                        </td>
+                        <td class="submitContentLabel">
+                          Split
+                        </td>
+                        <td class="submitContentLabel">
+                          Total Value (USD)
+                        </td>
+                      </tr>
+                      <tr repeat.for="daoplomat of submitData.terms.daoplomatRewards.daoplomats">
+                        <td>
+                          <etherscanlink
+                            address.bind="daoplomat.address"
+                            text="${daoplomat.address | smallHexString}">
+                          </etherscanlink>
+                        </td>
+                        <td>${daoplomat.rewardSplitPercentage / submitData.terms.daoplomatRewards.percentage * 100}%</td>
```

Comment:
is this the conversion logic you have somewhere else in the view model?

Reply:
I have it in two other places but in Typescript